bin/k8s-read-config

```diff
@@ -1,15 +1,23 @@
 #!/bin/bash
 
+LOG_LEVEL="${LOG_LEVEL:-info}"
+
+rok8s_echo() {
```

Comment:
Interesting idea.  A thought for another day - how long until we end up moving this to a real language so that we don't have to implement things like logging ourselves?

Reply:
Yeah very interested in writing a _much_ simpler tool with a proper language that could support our use cases moving forward.